packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCS/SampleModuleCS.cs

```diff
@@ -102,5 +106,30 @@ public void ExplicitPromiseMethodWithArgs(double arg, ReactCallback<double> reso
         }
 
         #endregion
+
+        #region Events
+
+        [ReactEvent]
+        public ReactEvent<int> TimedEvent { get; set; }
+
+        [ReactMethod]
+        public int StartTimedEvent(int intervalMS)
+        {
+            Debug.WriteLine($"{Name}.{nameof(StartTimedEvent)}({intervalMS})");
+
+            int id = _timers.Count;
+
+            _timers.Add(ThreadPoolTimer.CreatePeriodicTimer(new TimerElapsedHandler((timer) =>
```

Comment:
>_timers [](start = 12, length = 7)

Do we need to stop timers explicitly?
Otherwise the lambda is capturing 'this' and thus the object is never deleted.

Reply:
Yes, fixed.